src/Vaxis.zig

```diff
@@ -167,6 +167,18 @@ pub fn exitAltScreen(self: *Vaxis) !void {
 /// capabilities will be delivered to the client and possibly intercepted by
 /// Vaxis to enable features
 pub fn queryTerminal(self: *Vaxis) !void {
+    try self.queryTerminalSend();
+
+    // 1 second timeout
+    std.Thread.Futex.timedWait(&self.query_futex, 0, 1 * std.time.ns_per_s) catch {};
+
+    try self.enableTerminalFeatures();
```

Comment:
This should be fixed to `enableDetectedFeatures` now :)

Reply:
Oops!